src/sdk/object_sdk.js

```diff
@@ -258,7 +258,7 @@ class ObjectSDK {
         });
     }
 
-    _setup_single_namespace(namespace_resource_config, bucket_id) {
+    _setup_single_namespace(namespace_resource_config, bucket_id, bucket) {
```

Comment:
We can pass this function only the bucket and get `bucket.namespace.write_resource` and `bucket._id` from it